src/main/java/com/gascalc/model/GasCalcModel.java

```diff
@@ -37,8 +57,36 @@ public void setCarModel(String carModel) {
 	public void setYear(int year) {
 		this.year = year;
 	}
+	public double getHighwayMPG() {
+		return highwayMPG;
+	}
+
+
+	public void setHighwayMPG(double highwayMPG) {
+		this.highwayMPG = highwayMPG;
+	}
+
+
+	public double getCityMPG() {
+		return cityMPG;
+	}
+
+
+	public void setCityMPG(double cityMPG) {
+		this.cityMPG = cityMPG;
+	}
+	
+	public void setId(int id) {
+		this.id = id;
+	}
 	
+	public int getId() {
+		return id;
+	}
+	@Override
 	public String toString() {
-		return "Your car Make: " + carMake + "Model: " + carModel + "Year: " + year;
+		return "GasCalcModel [id=" + id + ", carMake=" + carMake + ", carModel=" + carModel + ", year=" + year
+				+ ", highwayMPG=" + highwayMPG + ", cityMPG=" + cityMPG + "]";
```

Comment:
Generally I let the IDE generate the toString method automatically, otherwise you may have some unintended formating. The format for the original toString would like like "Your car Make: FordModel:FusionYear:2015" whereas the auto-generated ones generally automatically include delimiters. This isn't a huge deal, but in my opinion the more code you can offload to your IDE the better.